site/content/docs/manifest/backend-service.en.md

```diff
@@ -67,3 +67,27 @@ The architecture type for your service. [Backend Services](../concepts/services.
 {% include 'image-config.md' %}
 {% include 'image-healthcheck.md' %}
 {% include 'common-svc-fields.md' %}
+
+<div class="separator"></div>
+
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>      
+The `network` section contains parameters for connecting to AWS resources in a VPC.
+
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>    
+Subnets and security groups attached to your tasks.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-placement" href="#network-vpc-placement" class="field">`placement`</a> <span class="type">String</span>  
+Must be one of `'public'`, `'private'`, or `'isolated'`. Defaults to launching your tasks in public subnets.
```

Comment:
Should we update our doc with isolated given that this feature isn't fully working yet?

Reply:
Good point--these are now targeting a branch so everything can get fully working and pen tested before merging into mainline.